### PR TITLE
Allow customization of the shutdown command UPS/NUT uses to take down the system

### PR DESCRIPTION
**Pull request by snowzach**
In my case (and other's from what I have seen) it would nice to customize via the GUI the command that the NUT tools use to shutdown the system. I have a script that nicely shuts down all of the embedded VM's relying on my FreeNAS NFS share, then it shuts down FreeNAS that's running as a VM. 

This adds the option to customize the command to the GUI and puts it in the upsmon.conf.

### CODE REVIEW

File: gui/services/models.py [hunk 1]
```diff
@@ -1290,6 +1290,15 @@ class UPS(Model):
             "happens to come back before the time is up the "
             "shutdown is canceled."),
     )
+    ups_shutdowncmd = models.CharField(
+        max_length=255,
+        verbose_name=("Shutdown Command"),
```
Comment (williambr) [inline gui/services/models.py]: It is missing a underline for translation, _("Shutdown Command")

**Comment by snowzach** [comment]
Sorry about that.. I think it's fixed.. Sorry, this is my first pull request.. :-)

**Comment by williambr** [comment]
Thanks!

**Comment by snowzach** [comment]
Just curious... Since there is no bug or feature tied to this (that I know of) will I be able to tell when it's part of a release? Any idea when a target would be?

**Comment by williambr** [comment]
You're right, I forgot to do that. Do you mind creating a ticket referencing this bug so we can track it?

**Comment by snowzach** [comment]
Huh... It would appear that I do not have access to create an issue.. I can only create pull requests...

**Comment by williambr** [comment]
On bugs.freenas.org, not here.

**Comment by snowzach** [comment]
Oh... sorry, done.. Thanks!

**Pull request merged**